stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/ChannelsScreen.kt

```diff
@@ -141,6 +141,7 @@ public fun ChannelsScreen(
                     .align(Alignment.BottomCenter),
                 selectedChannel = selectedChannel!!,
                 currentUser = user,
+                isMuted = listViewModel.isChannelMuted(selectedChannel!!.cid),
```

Comment:
Same here. Worst case, we could use one of the capturing functions such as let, but I'd still prefer a simple capturing value.